### PR TITLE
1488 authorization

### PR DESCRIPTION
**Pull request by ledsoft**
https://kbss.felk.cvut.cz/redmine/issues/1488

### CODE REVIEW

File: src/component/hoc/withLoading.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import getDisplayName from "../../util/getDisplayName";
 import classNames from "classnames";
 
 export interface WithLoadingProps {
-    loading: boolean,
+    loading?: boolean,
```
Comment (psiotwo) [inline src/component/hoc/withLoading.tsx:7]: why loading is optional?

File: src/i18n/en.ts [hunk 2]
```diff
@@ -499,6 +500,7 @@ const en = {
         "annotator.legend.definition.tooltip": "Definition of a known term created by a user.",
         'annotator.legend.toggle.show': 'Show legend',
```
Comment (psiotwo) [inline src/i18n/en.ts]: You might want to replace single quotes here as well

**Review by psiotwo — APPROVED** [review]
lgtm, minor comment

File: src/component/hoc/withLoading.tsx (hunk 1 above)
Comment (ledsoft) [reply src/component/hoc/withLoading.tsx:7]: The HOC API was causing typing issues. The effect is the same so it was simpler than trying to figure out how the API should look like...

File: src/i18n/en.ts (hunk 2 above)
Comment (ledsoft) [reply src/i18n/en.ts]: Good point. Should simplify merging to KODI.

**Pull request merged**